src/main/scala/tech/cryptonomic/conseil/Lorre.scala

```diff
@@ -75,6 +75,53 @@ object Lorre extends App with TezosErrors with LazyLogging with LorreAppConfig w
     apiOperations
   )
 
+  /** Schedules method for fetching baking rights */
+  system.scheduler.schedule(lorreConf.rights.initDelay, lorreConf.rights.interval)(writeFutureRights())
+
+  /** Fetches future baking and endorsing rights to insert it into the DB */
+  def writeFutureRights(): Unit = {
+    import cats.implicits._
+
+    implicit val mat = ActorMaterializer()
+    logger.info("BER Fetching future baking and endorsing rights")
+    val blockHead = tezosNodeOperator.getBlockHead()
+    val brLevelFut = apiOperations.fetchMaxBakingRightsLevel()
+    val erLevelFut = apiOperations.fetchMaxEndorsingRightsLevel()
+
+    (blockHead, brLevelFut, erLevelFut).mapN { (head, brLevel, erLevel) =>
+      val headLevel = head.data.header.level
+      val rightsStartLevel = math.max(brLevel, erLevel) + 1
+      logger.info(s"BER Current levels block: $headLevel baking rights: $brLevel endorsing rights: $erLevel")
```

Comment:
Possibly better to tell in the log entry that the block level is the current tezos head, while the rights are the top level stored in conseil?

Reply:
changed log